tools/dependency/release_dates.py

```diff
@@ -14,17 +14,16 @@
 import os
 import sys
 import argparse
-
 import github
 
 import exports
 import utils
-
 from colorama import Fore, Style
 from packaging import version
 
 # Tag issues created with these labels.
 LABELS = ['dependencies', 'area/build', 'no stalebot']
+github_repo_location = "envoyproxy/envoy"
```

Comment:
as its a constant can you make it all caps, please

Reply:
Done